packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -938,7 +938,44 @@ export const components = ({
             margin: 0,
             marginInlineEnd: odysseyTokens.Spacing1,
           },
+          ...(ownerState.variant === "lamp" && {
+            paddingBlock: 0,
+            paddingInline: 0,
+            borderRadius: 0,
+            border: 0,
+            backgroundColor: "transparent",
+            color: odysseyTokens.TypographyColorBody,
 
+            "&::before": {
+              content: "''",
+              width: ".64em",
+              height: ".64em",
```

Comment:
Are these the styles we had previously thath you copied over?

Reply:
These were copied over, I don't love the magic number either though, I'll check it out to see if we can use a token